examples/integrations/cdi/pokemons/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2020 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+javax.sql.DataSource.test.dataSourceClassName=org.h2.jdbcx.JdbcDataSource
+javax.sql.DataSource.test.dataSource.url=jdbc:h2:mem:test
```

Comment:
See if you need to add [`;DB_CLOSE_DELAY=-1`](https://dba.stackexchange.com/a/224339/31635) to this for your scenario.